forge/ee/lib/alerts/index.js

```diff
@@ -0,0 +1,54 @@
+const { Op } = require('sequelize')
+
+const { Roles } = require('../../../lib/roles')
+
+module.exports = {
+    init: function (app) {
+        if (app.postoffice.enabled) {
+            app.config.features.register('emailAlerts', true, true)
+            app.auditLog.alerts = {}
+            app.auditLog.alerts.generate = async function (projectId, event) {
+                if (app.postoffice.enabled) {
+                    const project = await app.db.models.Project.byId(projectId)
+                    const settings = await app.db.controllers.Project.getRuntimeSettings(project)
+                    const emailAlerts = settings.emailAlerts
+                    let template
+                    if (emailAlerts?.crash && event === 'crashed') {
+                        template = 'Crashed'
+                    } else if (emailAlerts?.safe && event === 'safe-mode') {
+                        template = 'SafeMode'
+                    }
+                    if (!template) {
+                        return
+                    }
+                    const team = await app.db.models.Team.byId(project.Team.id)
+                    const where = {
+                        TeamId: team.id
+                    }
```

Comment:
At first glance, this appears to be an unneeded lookup of `Team`. I would expect `project.Team.id` to be the same as `team.id` - which is all it's using `team` for.

That said, given this feature is flagged against the teamType as well, I think this needs something like this (please test it first!)
```
if (!team.TeamType.getFeatureProperty('emailAlerts', false)) {
  return
}
```

Reply:
I've pushed an update, had to swap the team lookup for a TeamType lookup, but should cover all angles (e.g. when team type is changed to one without the option enabled)